### PR TITLE
Update the workflows to match covalent devops

### PR DESCRIPTION
**Pull request by venkatBala**
- [ ] I have added the tests to cover my changes.
- [x] I have updated the documentation, VERSION, and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -45,36 +45,84 @@ jobs:
...
+
+      - name: Build SSH plugin
+        run: python setup.py sdist
+
+      - name: Validate plugin
+        run: |
+          VERSION="$(cat ./VERSION)"
+          cd dist
+          tar xzf covalent-ssh-plugin-${VERSION}.tar.gz
+          diff covalent-ssh-plugin-${VERSION}/covalent_ssh_plugin/ssh.py ../covalent_ssh_plugin/ssh.py
```
Comment (wjcunningham7) [inline .github/workflows/tests.yml:78]: this diff line doesn't do anything.  you should be comparing the directory contents.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -45,36 +45,84 @@ jobs:
...
+      - name: Start Covalent
         run: covalent start
+
       - name: Test plugin loading
         run: python -c "from covalent.executor import SSHExecutor"
+
       - name: Run tests
         run: |
-          PYTHONPATH=$PWD/tests pytest tests/ssh_tests.py -vv
+          PYTHONPATH=$PWD/tests pytest -vv tests/ssh_tests.py
```
Comment (wjcunningham7) [inline .github/workflows/tests.yml:95]: this should be running pytest on the tests directory, not on a single file

**Review by wjcunningham7 — CHANGES_REQUESTED** [review]

**Comment by wjcunningham7** [comment]
@venkatBala once this is merged into develop, open a PR which merges develop-covalent into develop.

File: .github/workflows/tests.yml (hunk 2 above)
Comment (venkatBala) [reply .github/workflows/tests.yml]: addressed

File: .github/workflows/tests.yml (hunk 1 above)
Comment (venkatBala) [reply .github/workflows/tests.yml]: updated the diff to compare directories

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [UNRELEASED] - 2022-05-26
```
Comment (wjcunningham7) [inline CHANGELOG.md:8]: this shouldn't have a date

**Review by wjcunningham7 — APPROVED** [review]

**Pull request merged**